root/usr/lib/systemd/system/enable-lldp

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enable LLDP on all interfaces
+After=lldpad.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/enable-lldp
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
This is missing `.service` extension.

Since you iterate over network ifaces, you need to make sure this service is started after network initialization (drivers are fully loaded), but before registration happens or proxy starts up:

```
After=network.target network-online.target
Before=foreman-proxy.service discovery-register.service
```